scripts/make_release_notes.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Converts github flavored markdown changelogs to release notes.
+"""
+
+import argparse
+import re
+import subprocess
+
+
+PRODUCTS = {
+    'Firebase/Auth/CHANGELOG.md': '{{auth}}',
+    'Firebase/Core/CHANGELOG.md': '{{core}}',
+    'Firebase/Database/CHANGELOG.md': '{{database}}',
+    'Firebase/DynamicLinks/CHANGELOG.md': '{{ddls}}',
+    'Firebase/InAppMessaging/CHANGELOG.md': '{{inapp_messaging}}',
+    'Firebase/InstanceID/CHANGELOG.md': 'InstanceID',
+    'Firebase/Messaging/CHANGELOG.md': '{{messaging}}',
+    'Firebase/Storage/CHANGELOG.md': '{{storage}}',
+    'Firestore/CHANGELOG.md': '{{firestore}}',
+    'Functions/CHANGELOG.md': '{{cloud_functions}}',
+
+    # 'Firebase/CoreDiagnostics/CHANGELOG.md': '?',
+    # 'Firebase/InAppMessagingDisplay/CHANGELOG.md': '?',
+    # 'GoogleDataTransport/CHANGELOG.md': '?',
+    # 'GoogleDataTransportCCTSupport/CHANGELOG.md': '?',
+    # 'GoogleUtilities/CHANGELOG.md': '?',
+    # 'Interop/CoreDiagnostics/CHANGELOG.md': '?',
```

Comment:
This is internal only, no need to have a changelog for it.

Reply:
Removed.